spec/podfile/target_definition_spec.rb

```diff
@@ -234,13 +234,13 @@ module Pod
 
       it 'allows depending on testspecs' do
         @parent.store_pod('RestKit', :testspecs => %w(Tests))
-        @parent.dependencies.map(&:name).sort.should == %w(RestKit/Tests)
+        @parent.dependencies.map(&:name).sort.should == %w(RestKit RestKit/Tests)
```

Comment:
Now this is correct! If I ask for the test spec 'Tests' I still want `RestKit` to be included.